projects/1.18/assets/goblins-dungeons/goblinsanddungeons/lang/zh_cn.json

```
@@ -0,0 +1,146 @@
+{
+	"itemGroup.goblinsanddungeons_item_group": "哥布林和地下城",
+	
+	"item.goblinsanddungeons.goblin_crown": "哥布林王冠",
+	"item.goblinsanddungeons.gob_spawn_egg": "哥布林刷怪蛋",
+	"item.goblinsanddungeons.hobgob_spawn_egg": "大型哥布林刷怪蛋",
+	"item.goblinsanddungeons.goblo_spawn_egg": "哥布落刷怪蛋",
+	"item.goblinsanddungeons.garch_spawn_egg": "加奇刷怪蛋",
+	"item.goblinsanddungeons.goom_spawn_egg": "古姆刷怪蛋",
+	"item.goblinsanddungeons.mimic_spawn_egg": "宝箱哥布林刷怪蛋",
+	"item.goblinsanddungeons.gobber_spawn_egg": "戈博刷怪蛋",
+	"item.goblinsanddungeons.ogre_spawn_egg": "食人魔刷怪蛋",
+	"item.goblinsanddungeons.goblin_king_spawn_egg": "哥布林之王刷怪蛋",
+	
+	"item.goblinsanddungeons.goblin_soul_bullet": "哥布林魂之弹",
+	"item.goblinsanddungeons.staff_amethyst": "法杖",
+	
+	"block.goblinsanddungeons.urn": "瓮",
+	"block.goblinsanddungeons.scoria": "熔渣石",
+	"block.goblinsanddungeons.scoria_bricks": "熔渣石砖",
+	"block.goblinsanddungeons.cracked_scoria_bricks": "裂纹熔渣石砖",
+	"block.goblinsanddungeons.chiseled_scoria": "錾制熔渣石砖",
+	"block.goblinsanddungeons.polished_scoria": "磨制熔渣石砖",
+	"block.goblinsanddungeons.scoria_slab": "熔渣石台阶",
+	"block.goblinsanddungeons.polished_scoria_slab": "磨制熔渣石台阶",
+	"block.goblinsanddungeons.scoria_brick_slab": "熔渣石砖台阶",
+	"block.goblinsanddungeons.scoria_stairs": "熔渣石楼梯",
+	"block.goblinsanddungeons.polished_scoria_stairs": "磨制熔渣石楼梯",
+	"block.goblinsanddungeons.scoria_brick_stairs": "熔渣石砖楼梯",
+	"block.goblinsanddungeons.scoria_wall": "熔渣石墙",
+	"block.goblinsanddungeons.polished_scoria_wall": "磨制熔渣石墙",
+	"block.goblinsanddungeons.scoria_brick_wall": "熔渣石砖墙",
+	
+	"block.goblinsanddungeons.ashed_soul_sand": "灰烬灵魂沙",
+	"item.goblinsanddungeons.soul_ash": "灵魂灰烬",
+	
+	"block.goblinsanddungeons.scoria_vertical_slab": "熔渣石竖直台阶",
+	"block.goblinsanddungeons.polished_scoria_vertical_slab": "磨制熔渣石竖直台阶",
+	"block.goblinsanddungeons.scoria_brick_vertical_slab": "熔渣石砖竖直台阶",
+	"block.goblinsanddungeons.chiseled_scoria_bricks": "裂纹熔渣石砖",
+	"block.goblinsanddungeons.scoria_pillar": "竖纹熔渣石",

```

```suggestion
	"block.goblinsanddungeons.scoria_pillar": "熔渣石柱",
```
`xx pillar`在1.16修改为`xx柱`
参见https://minecraft.fandom.com/zh/wiki/Minecraft_Wiki:%E8%AF%91%E5%90%8D%E6%A0%87%E5%87%86%E5%8C%96/%E5%8E%86%E5%8F%B2

---

感谢